README.md

```diff
@@ -87,9 +87,3 @@ Several tools compliment or duplicate Acquia CLI functionality. Acquia CLI can s
 ## Development / contribution
 
 Contributions to Acquia CLI are welcome subject to the [contributions policy](CONTRIBUTING.md), which also has more detailed information on how to develop Acquia CLI.
-
-## Support
-
-Until Acquia CLI has a stable 1.0.0 release support for it is limited to basic diagnostics only. Please refer to our
```

Comment:
You can keep `Please refer to our [Support Users Guide](https://docs.acquia.com/support/guide/) for more information.`